katas/011_sometimes_you_need_to_redirect_to_a_landing_page.md

```diff
@@ -0,0 +1,60 @@
+# Eleventh Kata: Sometimes You Need to Redirect to a Landing Page
+
+<i><a href="PLACEHOLDER: https://opensourceconnections.com/blog/2020/blah/blah" target="_BLANK">Read the blog, watch the video version of this Kata</a></i>
+
+Sometimes the right answer to a user's question isn't to actually run a search, but instead to drop them on the perfect page for them.  
+
+Redirect rules bypass the normal search results and redirect the page to a specific URL when the search query is entered. This is useful for terms that are very broad or for non-product searches.
+
+For example, a search for _computer_ can redirect to the computers category. Because the term computer is very broad, redirecting the user to the category page allows them to browse subcategories of computers to find a specific type of computer.
+
+Another example is a search for _employment_. Since an ecommerce site typically only indexes the product catalog, a redirect is necessary to send users to the careers page for this search.
+
+ __Active Search Management capability has been the biggest feature that commercial engines have had over the open source options for many years ;-)__
```

Comment:
I'm wondering if we should make it explicit that there are tools available now which place open source search engines on the same level as commercial ones, e.g.
'With open source tools like Querqy which is part of the Chorus stack, Lucene-based search engines have closed this gap for good.'

Reply:
Loveit!